msbuild/Makefile

```diff
@@ -326,17 +326,22 @@ $(foreach dll,$(MSBUILD_TASK_ASSEMBLIES),$(dll)/bin/$(CONFIG)/$(dll).dll): .buil
 .PHONY: $(MSBUILD_SYMLINKS)
 
 # Xamarin.MacDev.dll comes from the Xamarin.MacDev repository
-# Newtonsoft.Json.dll comes from a NuGet
```

Comment:
Can you provide QA a list of tasks that required `Newtonsoft.Json.dll` so they can make sure the changes are tested correctly ?